### PR TITLE
Add Apache 2 license badge and travis badge.

### PR DESCRIPTION
**Pull request by mrutkows**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# CocoaPods Podspecs for openwhisk-client-swift
+
+[![License](https://img.shields.io/badge/license-Apache--2.0-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0)
+#[![Build Status](https://travis-ci.org/apache/incubator-openwhisk-podspecs.svg?branch=master)](https://travis-ci.org/apache/incubator-openwhisk-podspecs)
```
Comment (jberstler) [inline README.md]: @mrutkows It appears that travis is not yet set up for this repo. The resulting badge just says "unknown" and clicking it takes me to a Travis page that says the repo is not active.

Comment (jberstler) [reply README.md]: Also, I don't think you need that leading `#`

**Review by jberstler — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mrutkows) [reply README.md]: @bjustin-ibm my fault, I was attempting to comment it out (duh... markdown) so it was there when I activated it.  Let me fix.

**Comment by mrutkows** [comment]
@bjustin-ibm correctly commented out line (for now) in a manner GitHub accepts.

**Review by jberstler — APPROVED** [review]
Thanks @mrutkows

**Pull request merged**